web-marketplace/src/graphql/indexer/AllTxes.graphql

```diff
@@ -0,0 +1,11 @@
+query IndexerAllTxes($first: Int, $offset: Int, $orderBy: [TxesOrderBy!]) {
```

Comment:
What are we using this for? It looks like we never use `getAllTxes`. Any reason for adding this now?

Reply:
This is the initial query provided by the indexer before we added `allEcocreditTxes`. After switching to `allEcocreditTxes` I kept the code for `allTxes` since we might still need it later on.

Reply:
Ok, sounds good. Just wanted to make sure I wasn't missing something.